packages/styles/settings-tools/_all-settings.scss

```diff
@@ -22,3 +22,5 @@
 @import './_s.select.scss';
 @import './_s.file-uploader.scss';
 @import './_s.flags.scss';
+@import './_s.stars.scss';
+@import './_s.hide.scss';
```

Comment:
I would put it on top because it is a dependency of components

Reply:
Fixed